src/main/java/org/aludratest/config/impl/SimplePreferences.java

```diff
@@ -29,11 +29,16 @@
  */
 public class SimplePreferences extends AbstractPreferences implements MutablePreferences {
 
-    private Map<String, String> values = new HashMap<String, String>();
+    private Map<String, String> values;
```

Comment:
Sonar's rule squid:S1258 wants a constructor that initializes the class' attributes.